src/test/groovy/net/fabricmc/loom/test/util/ProjectTestTrait.groovy

```diff
@@ -29,7 +29,7 @@ import org.gradle.testkit.runner.GradleRunner
 
 trait ProjectTestTrait {
 	final static String DEFAULT_GRADLE = "7.0.1"
-	final static String PRE_RELEASE_GRADLE = "7.2-20210527220045+0000"
+	final static String PRE_RELEASE_GRADLE = "7.2-20210612220215+0000"
```

Comment:
Looks like this has deprecated some stuff we are using, I will invistage this separately.